docs/source/python-library.rst

```diff
@@ -192,7 +192,7 @@ api
 
 .. automodule:: shapeflow.api
    :members:
-   :private-members: __id__
+   :private-members: __id__, _VideoAnalyzerDispatcher, _VideoAnalyzerManagerDispatcher, _DatabaseDispatcher, _FilesystemDispatcher, _CacheDispatcher
    :special-members: __id__
```

Comment:
Doesn't really help

Reply:
Add it in explicitly for now. Annoying since it becomes it's own section in `shapeflow.api`, but at least it's in the HTML now